src/main/java/com/google/api/generator/gapic/composer/rest/HttpJsonServiceCallableFactoryClassComposer.java

```diff
@@ -63,6 +77,15 @@ protected MethodDefinition createOperationCallableMethod(TypeStore typeStore) {
     String responseTemplateName = "ResponseT";
     List<String> methodTemplateNames =
         Arrays.asList(requestTemplateName, responseTemplateName, "MetadataT");
+
+    // Always add @BetaApi annotation to the generated createOperationCallable()method for now,
+    // until LRO is fully implemented.
+    AnnotationNode betaAnnotation =
+        AnnotationNode.withTypeAndDescription(
+            typeStore.get("BetaApi"),
+            "The surface for long-running operations is not stable yet and may change in the"
+                + " future.");
```

Comment:
Ditto - can we use a concrete termination criteria instead of sometime in the future?